pkg/controller/account/account_controller.go

```diff
@@ -72,7 +72,8 @@ const (
 	adminAccessArn = "arn:aws:iam::aws:policy/AdministratorAccess"
 	iamUserNameUHC = "osdManagedAdmin"
 
-	controllerName = "account"
+	controllerName   = "account"
+	accountFinalizer = "finalizer.aws.managed.openshift.io"
```

Comment:
this already exists, so no need to duplicate -> use `awsv1alpha1.AccountFinalizer`